package.json

```diff
@@ -31,8 +32,12 @@
     "eslint-plugin-promise": "^3.4.1",
     "eslint-plugin-standard": "^2.0.1",
     "gh-pages": "^1.0.0",
-    "git-clone": "^0.1.0",
-    "now": "^5.3.0"
+    "node-fetch": "^1.7.0",
+    "now": "^5.3.0",
+    "npmlog": "^4.1.0",
```

Comment:
I am fine with everything, just, we try to get rid of `npmlog` in the other JS repositories. How about not introducing it here? A simple `console.log()` or `process.stdout.write()` should do it here :)